packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -247,9 +248,33 @@ export class MessageWidget extends Component {
       ) {
         // Send notification of new message
         const avatars = props.avatar.get('items');
+        let cardsLength = 0;
+        let message = lastActivityFromThis.object.displayName;
+
+        if (Object.prototype.hasOwnProperty.call(lastActivityFromThis, 'object')
+        && Object.prototype.hasOwnProperty.call(lastActivityFromThis.object, 'cards') && lastActivityFromThis.object.cards.length > 0) {
+          cardsLength = lastActivityFromThis.object.cards.length;
+          const sharedCardsNotificationFormat = formatMessage(messages.sharedCards, {cardsLength});
```

Comment:
It might make it more readable for this variable to be named `sharedCardsNotificationMessage` instead.